src/components/Catalog/catalog.css

```diff
@@ -0,0 +1,98 @@
+a {
+  text-decoration: none;
+}
+li {
+  list-style: none;
+}
+body {
+  font-family: "Montserrat", sans-serif;
+}
+/* Courses Catalog */
+.courses-catalog {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  gap: 30px;
+  padding: 20px 10%;
+}
+.catalog-content {
+  grid-column: span 12;
+  text-align: center;
+  /* margin-bottom: 20px; */
+}
+.catalog {
+  grid-column: span 3;
+}
+.course,
+.course-explore {
+  margin-bottom: 15px;
+  font-weight: 700;
+}
+.course-text {
+  font-weight: 500;
+  font-size: 14px;
+}
+.yellow-text {
+  color: rgb(238, 241, 38);
+}
+.catalog {
+  padding: 20px;
+  background: rgba(245, 245, 220, 0.779);
+  transition: 0.5s;
+}
```

Comment:
- [ ] It is required to produce a close replica of what we have on the figma design. This includes color, font, typograph, background color, layout. The background color on your design does not tally to what we have on the figma. Kindly check the color and apply accordingly. This is the color use for the cards `#F6F2F2`

### Your Design

![your-design](https://github.com/Qoosim/practicode-frontend/assets/26597432/d9317c68-a1d1-4aa2-bc66-1f6b18571583)

### Figma Design

![figma-design](https://github.com/Qoosim/practicode-frontend/assets/26597432/2496de1f-9dad-4375-b18d-aad7b8e792aa)